FindImportProfile/FindImportProfileContainer/AbstractContainer.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  get,
+  noop,
+} from 'lodash';
+import { FormattedMessage } from 'react-intl';
+
+import { StripesConnectedSource } from '@folio/stripes/smart-components';
+
+import { listTemplate } from '@folio/data-import/src/components/ListTemplate';
+
+const idPrefix = 'uiPluginFindImportProfile-';
+
+export class AbstractContainer extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.logger = props.stripes.logger;
+    this.log = this.logger.log.bind(this.logger);
+  }
+
+  componentDidMount() {
+    this.source = new StripesConnectedSource(this.props, this.logger);
+    this.props.mutator.query.replace('');
+  }
+
+  componentDidUpdate() {
+    this.source.update(this.props);
+  }
+
+  onNeedMoreData = () => {
+    if (this.source) {
+      this.source.fetchMore(this.props.profileShape.RESULT_COUNT_INCREMENT);
+    }
+  };
+
+  querySetter = ({
+    nsValues,
+    state,
+  }) => {
+    if (/reset/.test(state.changeType)) {
```

Comment:
I would store this into variable to give more context.

Reply:
Done